website_docs/manual.md

```diff
@@ -7,9 +7,88 @@ aliases: [/docs/instrumentation/go/instrumentation, /docs/instrumentation/go/man
 
 Instrumentation is the process of adding observability code to your application. There are two general types of instrumentation - automatic, and manual - and you should be familiar with both in order to effectively instrument your software.
 
+## Initializing a Tracer
+
+To create spans, you'll need to initialize a tracer first. Some important things to consider:
+
+* Tracers should live at the module level and have a lifecycle congruent to an app's lifecycle
+* There should only be one tracer per module
+* Tracers need to be shut down when a program ends
+* You'll need an [exporter]({{< relref "exporting_data" >}}) to create a Tracer
```

Comment:
```suggestion
* You'll need an [exporter]({{< relref "exporting_data" >}}) to create a `TracerProvider`
```

Though even this isn't true.  The collector's `zpages` component uses a `TracerProvider` that currently does not export but does have an associated `SpanProcessor` that enables the `zpages` component to show in-flight traces.

Reply:
Are there other user scenarios that involve initializing a tracer where they don't want traces to export anywhere?

Reply:
`Tracer` or `TracerProvider`?  The `Tracer` is an API construct to be used by instrumentation without regard for, or even knowledge of, whether or how an SDK is configured.  An API `TracerProvider` is used similarly by instrumentation but an SDK `TracerProvider` must be intentionally configured by the application author/operator.  There are probably other situations where an application author would want to initialize an SDK `TracerProvider` without a `SpanProcessor` that results in exporting span data, though I can't enumerate them off the top of my head.